### PR TITLE
Make `Badge` priority class always present like in all other components

### PR DESCRIPTION
**Pull request by adamkudrna**
⚠️  This change affects output HTML and may break downstream snapshot tests.

### CODE REVIEW

**Review by atmelmicro — DISMISSED** [review]
`src/components/Button/helpers/getRootPriorityClassName.js` is not used anymore, afaik it is safe to delete

---

Snímek obrazovky 2024-09-03 v 13 15 58
Filled badges do not have the priority class

**Review by bedrich-schindler — APPROVED** [review]

File: src/components/Badge/Badge.jsx [hunk 1]
```diff
@@ -16,7 +17,7 @@ export const Badge = ({
     {...transferProps(restProps)}
     className={classNames(
       styles.root,
-      priority === 'outline' && styles.isRootPriorityOutline,
+      getRootPriorityClassName(priority, styles),
```
Comment (bedrich-schindler) [inline src/components/Badge/Badge.jsx:20]: CSS classes for priority=filled are missing in Badge.module.scss.

**Review by bedrich-schindler — CHANGES_REQUESTED** [review]

File: src/components/Badge/Badge.jsx (hunk 1 above)
Comment (bedrich-schindler) [reply src/components/Badge/Badge.jsx:20]: I got confused as @atmelmicro. While CSS classes are missing in this pull request, their are added in following pull request: https://github.com/react-ui-org/react-ui/pull/562

@adamkudrna, if you can, leave the information about it somewhere (probably in description). To be honest, it should be one PR as this leave unfinished feature in the code.

Closing comment and approving PR.

**Review by bedrich-schindler — APPROVED** [review]

File: src/components/Badge/Badge.jsx (hunk 1 above)
Comment (adamkudrna) [reply src/components/Badge/Badge.jsx:20]: Of course you are right, sorry, I didn't notice it. I tried to split my work into as many PRs as possible and this thing leaked 🙂.

BTW, moments like this make me feel we'd appreciate visual tests here in RUI.

Comment (adamkudrna) [reply src/components/Badge/Badge.jsx:20]: **UPDATE:** OK, I checked the rendered output and guess what, it looks correct 🎉. So what's happening here is that calling of a non-existing class name in CSS modules is not reported. When I think of it, I should have added the priority class check to tests and then I would know. 🤔 

obrazek

Anyway, thanks to both of you for catching this. I'll fix it and add a test (and then probably use my admin power to merge it so the PR cascade gets smaller).

Comment (adamkudrna) [reply src/components/Badge/Badge.jsx:20]: Didn't find anything, maybe the classes are just resolved differently in the test environment (which sounds terribly), I don't know 🤷🏻‍♂️.

Anyway, I added the missing class to CSS.

Comment (bedrich-schindler) [reply src/components/Badge/Badge.jsx:20]: If tests does not fail when they should, consider creating issue to look at it.

**Pull request merged**